src/main/cc/ConfEcalc/CMakeCache.txt

```diff
@@ -0,0 +1,376 @@
+# This is the CMakeCache file.
```

Comment:
All of these CMake outputs definitely don't need to be committed!
Try doing an [out of source](https://gitlab.kitware.com/cmake/community/-/wikis/FAQ#what-is-an-out-of-source-build) build to keep the source tree clean. Whatever directory you choose for your builds should go in the `.gitignore` file for the ConfEcalc folder. I've apparently been using something that starts with `cmake-`, but feel free to use whatever path is easy for you.